homeassistant/components/huawei_lte/config_flow.py

```diff
@@ -22,12 +22,11 @@
 from homeassistant.components.ssdp import ATTR_HOST, ATTR_NAME, ATTR_PRESENTATIONURL
 from homeassistant.const import CONF_PASSWORD, CONF_RECIPIENT, CONF_URL, CONF_USERNAME
 from homeassistant.core import callback
-from .const import CONNECTION_TIMEOUT, DEFAULT_DEVICE_NAME
 
 # https://github.com/PyCQA/pylint/issues/3202
```

Comment:
This comment belongs to the import below this one, and is now misplaced.

Reply:
Hmm, `isort` seems to **really** want to move it.

I improved the comment. Another (hacky) solution would be to add `isort:skip` to the other line...